### DISCUSSION TITLE
FunctionFlip - make AirPlay devices available for SpotiyConnect

### DISCUSSION THREAD

**Discussion by totti4ever** [philippe44/AirConnect#420] [General]
Hey, there once was an Android App turning AirPlay devices into spotify connect devices (Hify). As this project here seems to be technically similar, but not fenced to Smartphones, I was wondering whether you, @philippe44, could also offer something like Hify for Unix systems.

This would be absolutely incredible! If anybody knows such a project, please let me know :-)

**Comment by philippe44** [comment]
Yes with cspot now available this is doable but this is not something I'm interested into. I mean I don't have a need for that and you must have some for such projects to work.

**Comment by philippe44** [comment]
I might in fact do a UPnP to Spotify bridge (offering a UPnP device to be visible as a Spotify target). Not sure for AirPlay

**Comment by philippe44** [comment]
It's done in SpotConnect
